sbc_compassion/models/import_letters_history.py

```diff
@@ -62,7 +54,7 @@ class ImportLettersHistory(models.Model):
         'import.letter.line', 'import_id', 'Files to process')
     letters_ids = fields.One2many(
         'sponsorship.correspondence', 'import_id', 'Imported letters',
-        readonly=True)
+        readonly=True, ondelete='cascade')
```

Comment:
Not sure for this one. I would have put it for the field import_line_ids but not letters_ids.